docs/cpp/config-clang-mac.md

```diff
@@ -115,37 +115,32 @@ Remember, the C++ extension uses the C++ compiler you have installed on your mac
 
    ![Build task](images/clang-mac/default-build-task.png)
 
-You'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file.
+You'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file. Since we are using features from C++17 in this example, but the default C++ version in macOS is C++98, the first time you attempt to build, it will fail with Error "expected ';' at end of declaration gcc [Ln 9, Col 23]".
```

Comment:
```suggestion
You'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file. Because this example uses C++17, but the default C++ version in macOS is C++98, the first build attempt will fail with Error `expected ';' at end of declaration gcc [Ln 9, Col 23]`.
```

wouldn't it be better code formatting here?